benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -61,6 +60,33 @@ def _create_vm_scale(self, vm_num: str):
         self._oc.create_async(yaml=os.path.join(f'{self._run_artifacts_path}', f'{self._name}_{vm_num}.yaml'))
         self._oc.wait_for_vm_status(vm_name=f'{self._workload_name}-{self._trunc_uuid}-{vm_num}', status=VMStatus.Stopped)
 
+    def _run_vm(self):
+        """
+        This method run vm
```

Comment:
runs

This comment isn't helpful as it simply duplicates the name of the method; there's no point in even having it here at all.  If you want to explain it here do so, but a comment "because there's supposed to be a comment on each method" is gratuitous.